lib/linguist/languages.yml

```diff
@@ -2412,6 +2412,17 @@ Go Module:
   tm_scope: go.mod
   ace_mode: text
   language_id: 947461016
+Godot Resource:
+  type: data
+  color: "#355570"
+  extensions:
+  - ".tres"
+  - ".tscn"
+  filenames:
+  - project.godot
```

Comment:
We can also add:

* Extension `".gdns"` (for GDNative)
* Extension `".gdnlib"` (for GDNative)
* Extension `".gdextension"` (for GDExtension)
* File name `override.cfg` (might be too generic of a name to put in Linguist, but we can determine if it's for Godot or not if it's next to `project.godot` in the file system, idk how to do this in Linguist).

Reply:
Unfortunately there is nothing that will always appear in `override.cfg`, so yeah we can just leave it out.